hazelcast/src/test/java/com/hazelcast/map/EntryProcessorOffloadableTest.java

```diff
@@ -874,7 +874,7 @@ public void run() {
             }
         };
 
-        final int secondsToRun = 8;
+        final int secondsToRun = 55;
```

Comment:
This, together with the fact that `EntryProcessorOffloadableTest` is `@Parameterized`, means that running the test class now takes 6 minutes instead of 1 in the PR builder. Is it still a `@QuickTest`?

What about stopping the `TimeConsumingOffloadableTask` somehow once a heartbeat arrives to finish early? Or maybe rewriting the test to execute `TimeConsumingOffloadableTask` in a loop (a la `assertEventually`) until a heartbeat is detected (but that would be a different, weaker test).

Reply:
I considered the SlowTests category too, but I thought it would make the test weaker (by repeating it less frequently), so kept it as is for now to see if it fails again. On the other hand, the 6mins total duration, is a problem, so I don't know. If I move it as a SlowTest, then we might as well revert the timeouts to previous values.
Let me re-consider the options, and maybe try your idea of stopping it earlier (1st heartbeat).

Reply:
Its now a `SlowTest` and has a fast-exit when the 1st HB is received. Best case scenario it takes roughly 15s for 1 execution to complete.